packages/core/src/components/pill/pill.vars.scss

```diff
@@ -0,0 +1,41 @@
+@use '../../styles/vars/colours';
```

Comment:
This whole file needs to not use the colours scss file. It needs to use the css variables from the theme.css file so that it matches how we do colours in the reset of the DS